index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+  
+    <header class="header">
+      <h3 class="head">Welcome</h3>
+      <nav class="menu-bar">
+        <img src="images/Normal Button/Enabled.png" alt="">
+      </nav>
+    </header>
+    <section class="head-section">
+    <div class="hey-there">
+<h2 class="hey-text">Hey there. I’m Mariane
+ <p class="hey-text2"> I’m a software developer</p></h2>
+    </div>
+    <div class="text1">
+      <p class="text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+      <ul class="icons">   
+         <a href="" class="icon"> <img src="images/icons/twitter.svg" alt=""></a>  
+        <a href="" class="icon"> <img src="images/icons/i.png" alt="">
+          <img src="images/icons/n.png" alt="">
+         </a> 
+        <a href="" class="icon">  <img  class="medium" src="images/icons/medium@2x.png" alt=""></a>
+        <a href="" class="icon"><img src="images/icons/github.svg" alt=""></a>
+        <a href="" class="icon"><img src="images/icons/angellist.svg" alt=""></a> 
+      </ul>
```

Comment:
- [ ] Your social links are great, but to use the `ul` tag properly, the `li` tags should wrap up each list item. Kindly wrap each `a` tag with the `li` tag for better HTML semantics.